pkgs/tools/package-management/nix/default.nix

```diff
@@ -17,8 +17,8 @@ common =
   , stateDir
   , confDir
   , withLibseccomp ? lib.any (lib.meta.platformMatch stdenv.hostPlatform) libseccomp.meta.platforms, libseccomp
-  , withAWS ? stdenv.isLinux || stdenv.isDarwin, aws-sdk-cpp
-
+  , withAWS ? !enableStatic && (stdenv.isLinux || stdenv.isDarwin), aws-sdk-cpp
```

Comment:
Probably best to keep these options separate. There’s no reason for aws to not work statically, but it is broken currently.

Reply:
Since it can't compile it seems better to do it in the package rather than at call time.